keps/sig-network/1669-graceful-termination-local-external-traffic-policy/README.md

```diff
@@ -178,233 +183,91 @@ Yes, there will be unit tests in kube-proxy with the feature gate enabled and di
 
 ### Rollout, Upgrade and Rollback Planning
 
-<!--
-This section must be completed when targeting beta to a release.
--->
-
 ###### How can a rollout fail? Can it impact already running workloads?
 
-<!--
-Try to be as paranoid as possible - e.g., what if some components will restart
-mid-rollout?
--->
-
-TBD for beta.
+Roll out can fail if there are pods receiving traffic during termination but are unable to handle it.
+This can be unexpected, however, it is not any worse than existing behavior where kube-proxy would simply
+blackhole that traffic.
 
 ###### What specific metrics should inform a rollback?
 
-<!--
-What signals should users be paying attention to when the feature is young
-that might indicate a serious problem?
--->
-
-TBD for beta.
+Application-level metrics should be used to determine if traffic received during termination is causing issues.
```

Comment:
That's far from ideal, as often cluster admins may not understand application-level metrics.

I agree it might be hard to reflect the exact user-oriented behavior, but can we e.g. at least expose some kube-proxy level metric that will be showing how many:
(a) terminating & ready
(b) terminating & not-ready
endpoints can in theory be targetted?
[i.e. counters]